app/main.py

```diff
@@ -107,14 +103,13 @@ async def get_model_forecasts(
 
 
 @app.post('/models/{model}/forecasts/summaries/', response_model=schemas.WeatherForecastModelSummaryResponse)
```

Comment:
Right now the it's been hard coded to return the collated data for the last 5 days, but really we could pass any date range to get summaries for that date range - I'd rather stick with the generic name I've given it for that reason - though I agree /models/{model}/forecasts/historic would be more accurate. I wouldn't want to drop "forecasts" though - since there are potentially many different things we could be doing on /models/{model}.

Let's have a chat?